0.old/roles/integrator/faq.md

```diff
@@ -34,7 +34,7 @@ Our plan is to have a transparent Bug Bounty program with clear guidelines for p
 
 ### What contracts should we be aware of right now? {#what-contracts-should-we-be-aware-of-right-now}
 
-We have developed a number of [initial contracts](https://github.com/near/initial-contracts) with **ones in bold** being most mature at time of writing
+We have developed a number of [initial contracts](https://github.com/near/core-contracts) with **ones in bold** being most mature at time of writing
```

Comment:
should we just go ahead to rename this entirely to core contracts?

```suggestion
We have developed a number of [core contracts](https://github.com/near/core-contracts) with **ones in bold** being most mature at time of writing
```

Reply:
the files in this `0.old` folder are just left overs from the old documentation site, so there's no real need to update them. (these articles are not deployed to docs.near.org)